src/modules/components/member/memberDataListItem/memberDataListItemStructure/memberDataListItemStructure.tsx

```diff
@@ -0,0 +1,82 @@
+import { getAddress } from 'viem';
+import { useAccount } from 'wagmi';
+import { DataList, Heading, NumberFormat, Tag, formatterUtils, type IDataListItemProps } from '../../../../../core';
+import { truncateEthAddress } from '../../../../../utils/truncateEthereumAddress';
+import { MemberAvatar } from '../../memberAvatar';
+
+export interface IMemberDataListItemProps extends IDataListItemProps {
+    /**
+     * Whether the member is a delegate of current user or not.
+     */
+    isDelegate?: boolean;
+    /**
+     * The number of delegations the member has from other members.
+     */
+    delegationCount?: number;
+    /**
+     * The total voting power of the member.
+     */
+    votingPower?: number;
+    /**
+     * ENS name of the user.
+     */
+    ensName?: string;
+    /**
+     * 0x address of the user.
+     */
+    address: string;
+    /**
+     * Direct URL src of the user avatar image to be rendered.
+     */
+    avatarSrc?: string;
+}
+
+export const MemberDataListItemStructure: React.FC<IMemberDataListItemProps> = (props) => {
+    const { isDelegate, delegationCount = 0, votingPower = 0, avatarSrc, ensName, address, ...otherProps } = props;
+
+    const { address: currentUserAddress, isConnected } = useAccount();
+
+    const isCurrentUser = isConnected && address && currentUserAddress === getAddress(address);
+
+    const resolvedUserHandle = ensName != null && ensName !== '' ? ensName : address ?? undefined;
```

Comment:
Fallback to `undefined` is not needed, right?
```suggestion
    const resolvedUserHandle = ensName != null && ensName !== '' ? ensName : address;
```